experiment.py

```diff
@@ -0,0 +1,79 @@
+import asyncio
```

Comment:
I think we can add this to the `examples` folder, it's very useful to see the utility in action!